client/app/dedicated/server/translations/Messages_fr_FR.xml

```diff
@@ -1202,4 +1202,6 @@
    <translation id="server_rendezvous_profit" qtlid="521379">Profitez-en !</translation>
    <translation id="server_rendezvous_book" qtlid="521392">Programmer un rendez-vous</translation>
    <translation id="server_rendezvous_stop_bother" qtlid="521405">Ne plus montrer</translation>
+   <translation id="server_usage_chart_yaxis_label">Utilisation en%</translation>
+   <translation id="server_usage_chart_xaxis_label">Temps</translation>
```

Comment:
What's the unit ? Seconds ? Minutes ? Tell me and I'll give you the translation

Reply:
its dynamic based on what user has selected, last 12 hours, last 7 days, last 30 days and las 12 months. It can be hours or date

Reply:
its a time stamp

Reply:
Okay so "Temps" should be good then :)